src/components/WalletConnectWrapper/WalletConnectWrapper.types.ts

```diff
@@ -0,0 +1,5 @@
+export type WalletConnectInfoProps = {
+  expectedChainName?: string;
```

Comment:
I'll try to finish the Agregator component today. Then you can use `ChainEnum` here